plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/basic-setup.spec.js

```diff
@@ -1,22 +1,24 @@
 const { test, expect } = require( '@playwright/test' );
+const { lstat } = require('fs');
+const { getTranslationFor } = require( './../../test-data/data' );
 
 test.describe( 'Store owner can finish initial store setup', () => {
 	test.use( { storageState: process.env.ADMINSTATE } );
 	test( 'can enable tax rates and calculations', async ( { page } ) => {
 		await page.goto( 'wp-admin/admin.php?page=wc-settings' );
 		// Check the enable taxes checkbox
 		await page.locator( '#woocommerce_calc_taxes' ).check();
-		await page.locator( 'text=Save changes' ).click();
+		await page.locator( `text=${getTranslationFor('Save changes')}` ).click();
 		// Verify changes have been saved
 		await expect( page.locator( '#woocommerce_calc_taxes' ) ).toBeChecked();
 	} );
 
 	test( 'can configure permalink settings', async ( { page } ) => {
 		await page.goto( 'wp-admin/options-permalink.php' );
 		// Select "Post name" option in common settings section
-		await page.locator( 'label >> text=Post name' ).check();
+		await page.locator( `label >> text=${getTranslationFor('Post name')}` ).last().check();
```

Comment:
For this one, in some languages, if we do not select the last occurrence then the locator resolves to mulitple instances. For example, using .last(), in arabic::

<img width="654" alt="image" src="https://github.com/woocommerce/woocommerce/assets/105309450/da95b6db-a685-49e7-bf3d-26742bd46a0e">


<img width="803" alt="image" src="https://github.com/woocommerce/woocommerce/assets/105309450/e774ad8c-e16f-4af0-a740-910a388786e3">

However, if we do not use .last(), we see:

<img width="1533" alt="image" src="https://github.com/woocommerce/woocommerce/assets/105309450/f19b91d0-8f62-4509-b35d-b97a165e5688">

Reply:
This is because we have 2 sections on the same page with the same locator we filter by and `label >> text=${getTranslationFor('Post name')}` resolve for both of them, isn't it?

In that case, wouldn't it be better just to improve the locator so that we match the section we want? For example:
```
await page.locator( `sectionLocator >> label >> text=${getTranslationFor('Post name')}` ).last().check();
```

Is this possible? This way we would make the test more deterministic and less susceptible of flakiness, because it wouldn't fail for example if we add a new element that resolves to `label >> text=${getTranslationFor('Post name')}` after the one we choose right now.

Reply:
Agreed! I will update the locator